src/betterproto/templates/template.py.j2

```diff
@@ -54,11 +52,11 @@ class {{ message.py_name }}(betterproto.Message):
     {% endif %}
 
     {% if message.deprecated or message.deprecated_fields %}
-    def __post_init__(self) -> None:
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
the `__init__` should have the correct argument list.

Reply:
That's gonna be a challenge. I'll see what I can do.

Reply:
Unless I'm missing something I think it should be fairly straight forward. If you provide a get_kwargs method on the relevant compiler class that formats the fields and their default values as a kwargs list, and then call it here and on line 59. Does that sound feasible?

Reply:
I've implemented it, it wasn't anywhere near as bad as I expected.